pkg/client/didexchange/client.go

```diff
@@ -96,15 +99,16 @@ func New(ctx provider) (*Client, error) {
 
 // CreateInvitation create invitation
 func (c *Client) CreateInvitation(label string) (*Invitation, error) {
-	verKey, err := c.wallet.CreateEncryptionKey()
+	logger.Debugf("In CreateInvitation about to call CreateKeyPairsCombo ...")
+	_, sigPubKey, err := c.wallet.CreateKeyPairsCombo()
 	if err != nil {
 		return nil, fmt.Errorf("failed CreateSigningKey: %w", err)
 	}
-
+	logger.Debugf("In CreateInvitation CreateKeyPairsCombo success")
```

Comment:
we shouldn’t treat debug log level like an execution tracer :)

Reply:
I will remove logger